### ISSUE TITLE
Add Rate Limiting to API

### ISSUE THREAD

**Issue by uzair-ali10** [integrateme-co/integrate-io#38]
2 or 1 Request per second

**Comment by brijeshshah13** [comment]
@uzair-ali10 what are you using nginx on production to route requests between dns and server?

**Comment by uzair-ali10** [comment]
> @uzair-ali10 what are you using nginx on production to route requests between dns and server?

Nope, we don't use Nginx as of now, rate-limiting should be implemented using Express Rate Limit package

https://www.npmjs.com/package/express-rate-limit
https://www.geeksforgeeks.org/how-to-integrate-express-rate-limit-in-node-js/

**Comment by brijeshshah13** [comment]
cool. please assign this to me. also, if not nginx, how are the requests being routed?

**Comment by uzair-ali10** [comment]
> cool. please assign this to me. also, if not nginx, how are the requests being routed?

Cloudflare
